openmdao/recorders/sqlite_recorder.py

```diff
@@ -4,18 +4,40 @@
 from sqlitedict import SqliteDict
 from openmdao.recorders.base_recorder import BaseRecorder
 from openmdao.util.record_util import format_iteration_coordinate
+from openmdao.recorders.recorders import IterationRecorder, MetadataRecorder
```

Comment:
same recorders.recorders problem here as well, in the interest of making it easy to find.